### PR TITLE
added directive for autofocus and removed docker

### LINKED ISSUE #200: Remove docker directory
We are not planning to use docker in the near future, so lets remove the `docker` directory and its dependencies.

### PR DESCRIPTION
**Pull request by anweshknayak**
fixed #200  #168 #199 #197

### CODE REVIEW

**Comment by aka-jain** [comment]
@anweshknayak :From now, try to make a different PR for Backened and Frontend.

**Comment by anweshknayak** [comment]
@aka-jain please review

**Comment by anweshknayak** [comment]
Sure @aka-jain

File: frontend/app.styles.json [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
+      "angular-animate",
+      "angular-ui-router",
+      "angular-aria",
+      "angular-message",
       "angular-material",
-      "materialize"
+      "materialize",
+      "lodash",
+      "ngSmoothScroll",
+      "ng-focus-if"
```
Comment (aka-jain) [inline frontend/app.styles.json]: @anweshknayak : can you add this directive to login and signup page?

Comment (anweshknayak) [inline frontend/app.styles.json]: added. sorry forgot to put it there

**Comment by aka-jain** [comment]
@anweshknayak Great work.
@deshraj: Frontend source is OK from my side. You can merge after reviewing Backend sorce.

File: frontend/app.scripts.json [hunk 2]
```diff
@@ -24,7 +25,8 @@
        "angular-material" : "bower_components/angular-material/angular-material.min.js",
        "materialize": "bower_components/materialize/dist/js/materialize.min.js",
        "lodash":"bower_components/lodash/dist/lodash.min.js",
-       "ngSmoothScroll":"bower_components/ngSmoothScroll/lib/angular-smooth-scroll.js"
+       "ngSmoothScroll":"bower_components/ngSmoothScroll/lib/angular-smooth-scroll.js",
+       "ng-focus-if":"bower_components/ng-focus-if/focusIf.min.js"
```
Comment (TroJan) [inline frontend/app.scripts.json]: can you please add a space after : 
also update for ngSmoothScroll

File: frontend/bower.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "materialize": "^0.97.8",
     "jquery": "^3.1.1",
     "lodash": "^4.17.2",
-    "ngSmoothScroll":"master"
+    "ngSmoothScroll":"master",
```
Comment (TroJan) [inline frontend/bower.json]: can you add here space here for both keys?

File: settings/common.py [hunk 4]
```diff
@@ -62,6 +62,8 @@
     'rest_framework.authtoken',
     'rest_framework',
     'rest_framework_docs',
+    'rest_framework_expiring_authtoken',
+    'autofixture'
```
Comment (TroJan) [inline settings/common.py]: `THIRD_PARTY_APPS` contains apps in alphabetical order except for `rest_framework.authtoken`.
Can you please add your apps in alphabetical order too?

Comment (anweshknayak) [inline settings/common.py]: Sure .

File: frontend/bower.json (hunk 3 above)
Comment (anweshknayak) [inline frontend/bower.json]: yep

**Comment by anweshknayak** [comment]
@TroJan My bad. Next time I will take care of it.

**Comment by anweshknayak** [comment]
@TroJan @deshraj please review

File: settings/common.py [hunk 5]
```diff
@@ -62,6 +62,7 @@
     'rest_framework.authtoken',
     'rest_framework',
     'rest_framework_docs',
+    'rest_framework_expiring_authtoken',
```
Comment (deshraj) [inline settings/common.py]: Can you also add rest_framework_expiring_authtoken in a new PR?

Comment (anweshknayak) [inline settings/common.py]: @deshraj done

**Comment by anweshknayak** [comment]
@deshraj please review

**Pull request merged**